app/javascript/components/automate-simulation-form/index.jsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState, useEffect } from 'react';
+import MiqFormRenderer, { useFormApi } from '@@ddf';
+import { FormSpy } from '@data-driven-forms/react-form-renderer';
+import PropTypes from 'prop-types';
+import { Loading, Button } from 'carbon-components-react';
+import { CheckmarkFilled, Close } from '@carbon/icons-react';
```

Comment:
Since you no longer are using this icons they should be deleted. Can you please remove these and fix the other lint errors in this file